Resources/Prototypes/Corvax/Entities/Objects/Consumable/Drinks/drinks_bottles.yml

```
@@ -0,0 +1,18 @@
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        reagents:
+        - ReagentId: Campari
+          Quantity: 100
+  - type: Sprite
+    sprite: Corvax/Objects/Consumable/Drinks/campari.rsi
+  - type: SolutionContainerVisuals
+    maxFillLevels: 10
+    fillBaseName: fill-
+  - type: Sealable
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)